app/views/pages/home.html.erb

```diff
@@ -1,81 +1,46 @@
 <%= stylesheet_link_tag 'pages' %>
```

Comment:
Why is it so wide?

![image](https://user-images.githubusercontent.com/51726526/122298443-250d9300-ced3-11eb-89a3-664410707261.png)

Reply:
Resolved at commit 90492df